example_projects/transfer/nlb_observability_stack/main.tf

```diff
@@ -0,0 +1,26 @@
+terraform {
+  required_providers {
+    doublecloud = {
+      source = "registry.terraform.io/doublecloud/doublecloud"
+    }
```

Comment:
should probably specify a version here to match the convention for AWS used below